bin/node/runtime/src/lib.rs

```diff
@@ -501,9 +501,11 @@ impl pallet_treasury::Trait for Runtime {
 	type ApproveOrigin = pallet_collective::EnsureMembers<_4, AccountId, CouncilCollective>;
 	type RejectOrigin = pallet_collective::EnsureMembers<_2, AccountId, CouncilCollective>;
 	type Event = Event;
-	type ProposalRejection = ();
+	type KtonProposalRejection = ();
+	type RingProposalRejection = ();
 	type ProposalBond = ProposalBond;
-	type ProposalBondMinimum = ProposalBondMinimum;
+	type RingProposalBondMinimum = ();
+	type KtonProposalBondMinimum = ();
```

Comment:
```
type KtonProposalBondMinimum = ();
```

?

Reply:
Add this line before because the outdated `StakingBalances` enum can not be `Get` from `u32`, now it's possible to use one `ProposalBondMinimum` for both `Ring` and `Kton`, but, independent `ProposalBondMinimum` might support more needs?

Reply:
Changed these to constant in the lastest commit.